app/controllers/OnlineTestController.scala

```diff
@@ -65,7 +65,7 @@ trait OnlineTestController extends BaseController {
   def getOnlineTest(userId: String) = Action.async { implicit request =>
```

Comment:
Agreed - we will go back and clean up these things once we have released as there are a few areas like this we need to tidy up but we need to agree as a team what the naming conventions will be.